mainFV2.cpp

```diff
@@ -122,34 +123,34 @@ int main(int argc, char** argv)
         Tbegin = std::chrono::steady_clock::now();
 
         std::vector<TargetBox> boxes;
-        yoloF2.detection(frame, boxes);
-        draw_objects(frame, boxes);
+        yoloF2.detection(frame, boxes); // Object detection and categorization is in this function (from yolo-fastestv2.cpp)
+        draw_objects(frame, boxes); // Draw the box and the category
         Tend = std::chrono::steady_clock::now();
 
         //calculate frame rate
         f = std::chrono::duration_cast <std::chrono::milliseconds> (Tend - Tbegin).count();
         if(f>0.0) FPS[((Fcnt++)&0x0F)]=1000.0/f;
         for(f=0.0, i=0;i<16;i++){ f+=FPS[i]; }
-        putText(frame, cv::format("FPS %0.2f", f/16),cv::Point(10,20),cv::FONT_HERSHEY_SIMPLEX,0.6, cv::Scalar(0, 0, 255));
-
-        //Change this
-        std::cout << "how many human: " << humans << std::endl;
-        std::cout << "delay: " << delay << std::endl;
-        std::cout << "dt: " << clock() - now << std::endl;
-        std::cout << "capacity: " << humans*100/36 << "%" << std::endl;
-
-        //Delay send
-        if (clock() - now > delay){
-            std::cout << "Delayed print" << std::endl;
-            ofstream MyFile("humans.txt");
-            MyFile << humans;
-            ofstream MyFile1("capacity.txt");
-            MyFile1 << humans*100 / 36 ;
-
-            MyFile.close();
-            MyFile1.close();
-
-            now = clock();
+        putText(frame, cv::format("FPS %0.2f", f/16),cv::Point(10,20),cv::FONT_HERSHEY_SIMPLEX,0.6, cv::Scalar(0, 0, 255));
+
+        //Change this
+        std::cout << "how many human: " << humans << std::endl;
+        std::cout << "delay: " << delay << std::endl;
+        std::cout << "dt: " << clock() - now << std::endl;
+        std::cout << "capacity: " << humans*100/full_capacity-1 << "%" << std::endl;
+
+        //Delay send
+        if (clock() - now > delay){
+            std::cout << "Delayed print" << std::endl;
+            ofstream MyFile("humans.txt");
+            MyFile << humans - 1;

```

Comment:
change to if humans > 0, humans - 1 for output

Reply:
or capacity + 1 for full capacity

Reply:
Noted, Thanks